src/templateAccessibilityElementsContentRule.ts

```diff
@@ -32,21 +32,22 @@ class TemplateAccessibilityAnchorContentVisitor extends BasicTemplateAstVisitor
 
 export class Rule extends Rules.AbstractRule {
   static readonly metadata: IRuleMetadata = {
-    description: 'Ensures that the anchor element has some content in it',
+    description: 'Ensures that the heading and anchor elements have content in it',
     options: null,
     optionsDescription: 'Not configurable.',
-    rationale: 'Anchor elements should have content to be accessible by screen readers',
-    ruleName: 'template-accessibility-anchor-content',
+    rationale: 'Heading and anchor elements should have content to be accessible by screen readers',
+    ruleName: 'template-accessibility-elements-content',
     type: 'functionality',
     typescriptOnly: true
   };
 
-  static readonly FAILURE_MESSAGE = 'Anchor element should have content';
+  static readonly FAILURE_MESSAGE = 'Heading elements should have content';
```

Comment:
Heading or Anchor

Reply:
@mohammedzamakhan would you address this comment?

Reply:
Done 👍